### DISCUSSION TITLE
Deno task files

### DISCUSSION THREAD

**Discussion by Conaclos** [denoland/deno#18292] [Ideas]
It could be nice to be able to write a multi-line task in a _task file_ instead of one-liner task in the deno configuration file.

For instance a file named `build`:

```
#!deno task

command1:
command2:
command3:
...
```

instead of:

```json
{
  "tasks": {
    "build": "command1; command2; command3; ..."
  }
}
```

The task file could be run as follows:

```sh
deno task ./build
```

Because we use a shebang, we could also write:

```sh
./build
```

**Comment by bartlomieju** [comment]
This was asked previously and our recommendation for this is to just write JS/TS script that needs to do a lot of commands and call that script from `deno task`, like so:
```
// build.js
command1;
command2;
command3;
```
```
{
  "tasks": {
    "build": "deno run build.js"
  }
}
```
